src/systemverilog/systemverilog.ts

```diff
@@ -255,7 +323,7 @@ export const language = <ILanguage>{
 			[/@encoding?R\"(?:([^ ()\\\t]*))\(/, { token: 'string.raw.begin', next: '@raw.$1' }],
 
 			// identifiers and keywords
-			[/[a-zA-Z_]\w*/, {
+			[/[a-z_]\w*/, {
```

Comment:
hey what does this change do?

Reply:
none of the system verilog keywords have capital letters, so just removing that!

Reply:
ok!